setup.py

```diff
@@ -72,6 +71,13 @@
             'codecov',
 
             'pytest-cov',
+        ],
+        ':python_version<"3.3"': [
```

Comment:
@jaraco We install `setuptools>=28.2` in Travis: https://github.com/cherrypy/cheroot/blob/6821264/.travis.yml#L156

Also Python 2.7.14 is shipped with `setuptools==28.8.0`: https://github.com/python/cpython/tree/v2.7.14/Lib/ensurepip/_bundled
(next patch release of 2.7.x should have https://github.com/python/cpython/tree/2.7/Lib/ensurepip/_bundled `>=39.0.1`)

So it looks okay to already use these features. (of course for envs where OS package manager controls things it might differ)